alf/bin/train.py

```diff
@@ -87,9 +85,17 @@ def train_eval(ml_type, root_dir):
 
 def main(_):
     FLAGS.alsologtostderr = True
+    random_seed = alf.get_config('TrainerConfig.random_seed')
```

Comment:
will this only read from the command line config?  What's the reason for making this change: first get random_seed, then set it and set config, then parse conf file?  It seems complicated and prone to error?

Reply:
moved to config_helpers.get_env() and commented there.